Makefile

```diff
@@ -23,6 +23,7 @@ USE_NINJA ?= 0
 FUZZ_BIN_DIR ?= ${ROOT_DIR}/afl-fuzz
 GCC_FUZZ_CC := ${FUZZ_BIN_DIR}/afl-gcc
 GCC_FUZZ_CXX := ${FUZZ_BIN_DIR}/afl-g++
+EMSCRIPTEN_DIR ?= $(dir $(shell which emcc))
```

Comment:
Will line 27 ever execute in this case? If not, we should remove it. Otherwise, lgtm. Thanks!